app/frontend/javascript/controllers/index.js

```diff
@@ -13,6 +13,9 @@ application.register("flashes", FlashesController)
 import HelloController from "./hello_controller"
 application.register("hello", HelloController)
 
+import NavbarToggleController from "./navbar_toggle_controller"
+application.register("navbar-toggle", NavbarToggleController)
+
```

Comment:
Did you create this controller with the rails generator ?
If not that means that everytime someone generates a new javascript controller it will delete this one from the index.js because it's not part of the stimulus manifest.

So if you haven't done so yet please run `rails g stimulus navbar_toggle` and transfer your code there if it's not done automatically

Reply:
I believe i did `rails g stimulus navbar_toggle` and it was auto-generated